src/Countries/Greece.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Greece extends Country
+{
+    public function countryCode(): string
+    {
+        return 'el';
+    }
+
+    /** @return array<string, string|CarbonImmutable> */
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'Πρωτοχρονιά'         => '01-01',
+            'Θεοφάνια'            => '01-06',
+            '25η Μαρτίου'         => '03-25',
+            'Πρωτομαγιά'          => '05-01',
+            'Δεκαπενταύγουστος'   => '08-15',
+            '28η Οκτωβρίου'       => '10-28',
+            'Χριστούγεννα'        => '12-25',
+            'Σύναξη της Θεοτόκου' => '12-26',
+
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function variableHolidays(int $year): array
+    {
+
+        $orthodox_easter = CarbonImmutable::createFromTimestamp(
+            $this->calculateOrthodoxEaster($year)
```

Comment:
You can use `orthodoxEaster()` in Country.php instead

Reply:
@Nielsvanpach The orthodoxEaster() is not calculating correct. Im getting wrong date of Easter.

not sure if this is working correct: easter_date($year, CAL_EASTER_ALWAYS_JULIAN);

In https://www.php.net/manual/en/function.easter-date.php gives the correct function to calculate the correct date + small fix. the same function I used in calculateOrthodoxEaster() 

Also Im getting error on other eastern orthodox countries. Orthodox Eastern for 2024 is in 5/5/2024 and the next day is holiday too 5/6/2024

Reply:
this is the correct way of calculate orthodox Easter

protected function calculateOrthodoxEaster(int $year): int
    {
      $a = $year % 4;
      $b = $year % 7;
      $c = $year % 19;
      $d = (19 * $c + 15) % 30;
      $e = (2 * $a + 4 * $b - $d + 34) % 7;
      $month = (int) (($d + $e + 114) / 31);
      $day = (($d + $e + 114) % 31) + 1;
      // julian to gregorian
      $jtg = (int) ($year / 100) - (int) ($year / 400) - 2;
      $easterDate = mktime(0, 0, 0, $month, $day + $jtg, $year);
      return (int) $easterDate;
    } // returns timestamp